examples/basic_openshift/conjur-java-api-example.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: conjur-cli-example
+  labels:
+    app: conjur-cli-example
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: conjur-cli-example
+  template:
+    metadata:
+      name: conjur-cli-example
+      labels:
+        app: conjur-cli-example
+    spec:
+      serviceAccountName: default
+      containers:
+      - image: cyberark/conjur-authn-k8s-client:latest
+        imagePullPolicy: IfNotPresent
+        name: authenticator
+        env:
+          - name: CONJUR_VERSION
+            value: "5"
+          - name: CONTAINER_MODE
+            value: sidecar
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: CONJUR_AUTHN_URL
+            value: https://conjur-oss.ofira-conjur-test.svc.cluster.local/authn-k8s/myauthenticator
+          - name: CONJUR_APPLIANCE_URL
+            value: https://conjur-oss.ofira-conjur-test.svc.cluster.local
+          - name: CONJUR_ACCOUNT
+            value: myaccount
+          - name: CONJUR_AUTHN_LOGIN
+            value: "host/conjur/authn-k8s/myauthenticator/apps/ofira-conjur-test/*/*"
+          - name: CONJUR_SSL_CERTIFICATE
+            valueFrom:
+              configMapKeyRef:
+                name: conjur-cert
+                key: ssl-certificate
+        volumeMounts:
+          - mountPath: /run/conjur
+            name: conjur-access-token
+      - name: ofira-conjur-java-client
+        image: docker-registry.default.svc:5000/ofira-conjur-test/ofira-conjur-java-client:latest
```

Comment:
we have a test Java app that we use with Summon (it's the [pet store demo](https://github.com/conjurdemos/pet-store-demo)) - maybe that could be updated to have a Java client flow so that we don't have to refer to your personal image in this example?